package.json

```diff
@@ -105,7 +105,6 @@
     "connect-redis": "^6.0.0",
     "cookie-session": "^1.4.0",
     "csurf": "^1.11.0",
-    "cypress-axe": "^0.13.0",
```

Comment:
`axe-core` is also listed as a dependency, can it be moved to devDependencies?

Reply:
Yes, missed that one. I'll move it.